pkg/porter/cnab.go

```diff
@@ -304,9 +306,14 @@ func (o *sharedOptions) defaultDriver() {
 // validateDriver validates that the provided driver is supported by Porter
 func (o *sharedOptions) validateDriver() error {
 	switch o.Driver {
-	case "docker", "debug":
+	case "docker", "debug", "kubernetes":
```

Comment:
I'm not sure why kubernetes was added in this PR? We haven't done any testing so I'm not sure if there's more work to be done on our side to make sure that we are properly loading and configuring that driver. Let's split that change into a separate PR and add an integration test for it.

Reply:
OK, I removed the Kubernetes driver.